app/api/routers/breeze_buddy/users/__init__.py

```
@@ -0,0 +1,205 @@
+    get_all_users_handler,
+    get_user_by_id_handler,
+    update_user_handler,
+)
+
+router = APIRouter()
+
+
+@router.post("/user", response_model=UserResponse, status_code=201)
+async def create_user_account(
+    user_data: UserCreate,
+    current_user: UserInfo = Depends(get_current_user_with_rbac),
```

Ruff (0.15.0)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)